airbyte-integrations/connectors/source-snowflake/source_snowflake/schema_builder.py

```diff
@@ -0,0 +1,44 @@
+from typing import Dict
+
+
+class SchemaTypes:
+    string: Dict = {"type": ["null", "string"]}
+    number: Dict = {"type": ["null", "number"]}
+    integer: Dict = {"type": ["null", "number"]}
+    boolean: Dict = {"type": ["null", "boolean"]}
+    date: Dict = {"type": ["null", "string"], "format": "date"}
+    array_with_any: Dict = {"type": ["null", "array"], "items": {}}
+    object: Dict = {"type": ["null", "object"]}
+
+
+mapping_snowflake_type_airbyte_type = {
```

Comment:
What about `variant`? I would expect some kind of datetime as well.

Does snowflake declare a list of all the types somewhere to make sure we don't miss some?

Reply:
what do you mean by variant ? 

Since it was said to keep the same behavior as before, I only mapped the types present here 
https://docs.airbyte.com/integrations/destinations/snowflake#data-type-map

Here is the exhaustive list of data types in snowflake
https://docs.snowflake.com/en/sql-reference/intro-summary-data-types

I can add the other types but I need the precise mapping between the new data types and the types in airbyte.

Reply:
Can you validate those types and get me a proposal for `timestamp`? 
![image](https://github.com/artefactory/airbyte/assets/3360483/7e1a1a0e-2b42-43ee-94e2-f2d99b5e7612)

Reply:
I validate the types and i will implement them. For timestamp, I will do some tests and come back to you.

Reply:
Cool, keep me posted!

Reply:
After some research, what looks the most representative for TIMESTAMP is timestamp_without_timezone since the global format is often YYYY-MM-DD HH:MM:SS. In postgres for example, there is no difference. What do you think about my proposition ?